src/blocks.rs

```diff
@@ -137,6 +137,8 @@ macro_rules! define_blocks {
 define_blocks!(
     amd_gpu,
     apt,
+    pacman,
```

Comment:
This list should be alphabetical so that it's easy to find the blocks.

Reply:
Done